oio/crawler/integrity.py

```diff
@@ -210,6 +213,28 @@ def complete_target_from_chunk_metadata(self, target, xattr_meta):
                                      "and container names: %s",
                                      cid, err)
 
+    def recover_and_complete_object_meta(self, target, chunk):
+        _, rawx_service, chunk_id = chunk.rsplit('/', 2)
+        # 1. Fetch chunk list from rdir (could be cached).
+        # Unfortunately we cannot seek for a chunk ID.
```

Comment:
Is it possible to use `start_after` with value `chunk_id - 1`?

Reply:
We can't use `start_after`... This parameter uses this format: `CONTAINER_ID|CONTENT_ID|CHUNK_ID`. And we don't know `CONTAINER_ID` and `CONTENT_ID` (because that's what we're looking for).

Reply:
In the future we can implement a route to make this filtering server-side.